README.md

```diff
@@ -65,12 +65,20 @@ To build the project:
 - Add dependency on Scala compiler plugin [Macro Paradise](https://github.com/scalamacros/paradise).
   - This can be done by adding `addCompilerPlugin("org.scalamacros" % "paradise" % "2.1.0" cross CrossVersion.full)` to the build settings in build.sbt.
   - An example project, which uses Macro Paradise can be found [here](https://github.com/scalamacros/sbt-example-paradise).
-- Add dependency on this repository.
-  - This can be done by adding the following line to the build.sbt file
-  ```scala
-  lazy val hiddenArgs = ProjectRef(uri("https://github.com/keddelzz/hidden-args.git"), "hiddenargs")
+- Add the following resolver to your `build.sbt`:
+```
+resolvers += Resolver.bintrayRepo("keddelzz", "maven")
```

Comment:
Also I recommend you to upload the library to JCenter. It's a _more_ central repository than your personal one, but not as bueracratized as Maven Central. It also works nicely with bintray (because the authors are the same).